infra/build/functions/build_project.py

```diff
@@ -308,10 +314,15 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-statements, to
     project_yaml,
     dockerfile,
     config,
-    additional_env=None):
+    additional_env=None,
+    use_caching=False):
   """Returns build steps for project."""
 
   project = Project(project_name, project_yaml, dockerfile)
+  if use_caching:
```

Comment:
hmm, I'm unsure if this works, because I think the build steps will generate a step to build the image. We need to remove that as well.